modules/mauth-authenticator/src/main/java/com/mdsol/mauth/RequestAuthenticator.java

```diff
@@ -47,21 +62,66 @@ public RequestAuthenticator(ClientPublicKeyProvider clientPublicKeyProvider, lon
     this(clientPublicKeyProvider, requestValidationTimeoutSeconds, new CurrentEpochTimeProvider());
   }
 
-  public RequestAuthenticator(ClientPublicKeyProvider clientPublicKeyProvider, long requestValidationTimeoutSeconds, EpochTimeProvider epochTimeProvider) {
+  /**
+   * Uses {@link com.mdsol.mauth.utils.ClientPublicKeyProvider} as the EpochTimeProvider
+   *
+   * @param clientPublicKeyProvider  PublicKey provider
+   * @param requestValidationTimeoutSeconds timeout
+   * @param disableV1
```

Comment:
please provide a description for `disableV1` param

Reply:
fixed https://github.com/mdsol/mauth-jvm-clients/pull/89/commits/639fbc9ed0fbf79345c5ddad83d1a8b379b68852